mason@suse.com

Document some of the reiserfs mount options

Index: linux.mm/Documentation/filesystems/reiserfsv3.txt
===================================================================
--- linux.mm.orig/Documentation/filesystems/reiserfsv3.txt	2003-09-23 13:59:22.000000000 -0400
+++ linux.mm/Documentation/filesystems/reiserfsv3.txt	2004-04-06 20:29:33.066719303 -0400
@@ -0,0 +1,55 @@
+ReiserFSv3
+
+Please visit http://www.namesys.com for reiserfs history and design
+information.
+
+Options
+=======
+
+When mounting a reiserfsv3 filesystem, the following options are
+accepted.  Defaults are marked with (*)
+
+tails=on        Pack partial blocks into the btree when the file is four
+                filesystem block or smaller, saving disk space
+
+tail=off        No tail packing
+
+tails=small*    Same as tails=on, but only do it for files using less
+                then one filesystem block.  Somewhat faster in many workloads
+
+notail          Same as tails=off
+
+conv            Convert a 3.5.x filesystem to a 3.6.x filesystem.  This is
+		a one way conversion, afterward you will not be able to
+		mount your filesystem under 2.2.x kernels any longer
+
+resize=NNN      Online resize the filesystem to NNN blocks.  This can
+                only be used to expand the filesystem
+
+jdev=device     Specify external journal location
+
+nolargeio       Don't export a large default IO size to userspace in the
+                stat call
+commit=NN       Set the commit interval to NN seconds (default=30)
+
+data=ordered*   Never commit new metadata until all data blocks it
+                references have been written.  This prevents stale disk
+                contents from being exposed after a crash
+
+data=writeback  No data ordering at all
+
+usrquota        Turn on user quotas
+
+grpquota        Turn on group quotas
+
+user_xattr      Enable extended attribute support
+
+nouser_xattr*   Disable extended attribute support
+
+acl             Enable access control lists
+
+noacl*          Disable access control lists
+
+Access control lists and extended attributes tools and documentation can
+be found at http://acl.bestbits.at
+
